Formula/mopidy-beets.rb

```diff
@@ -0,0 +1,27 @@
+class MopidyBeets < Formula
+  desc "Mopidy extension for playing music from Beets with its web plugin"
+  homepage "https://github.com/mopidy/mopidy-beets"
+  url "https://files.pythonhosted.org/packages/01/f1/b5104e2fdd1cce139804bb2298dd7cdc295f7a257386cc0302cf5d794586/Mopidy-Beets-3.1.0.tar.gz"
+  sha256 "bb0ad86c1ce70942a98a6d7e92efe71dbdbec411fbde7c5d8b3c745103a64a21"
+  head "https://github.com/mopidy/mopidy-beets.git"
+  revision 1
```

Comment:
This line should be removed. It is only present in some of the other formulas since they've gone through a large change to the formula setup without the upstream version changing. This makes the Homebrew version of this formula become "3.1.0_1" instead of "3.1.0".

```suggestion
```

Reply:
Done !